datahub-graphql-core/src/main/resources/search.graphql

```diff
@@ -1070,3 +1100,99 @@ type SystemFreshness {
   """
   freshnessMillis: Long!
 }
+
+"""
+Input required for browse queries
+"""
+input BrowseInputV2 {
```

Comment:
Tiny nit... by convention we always put Input at the very end: BrowseV2Input

Reply:
Not a major issue

Reply:
oh yeah for sure, will update